src/session.test.js

```diff
@@ -0,0 +1,73 @@
+/* @flow */
+import { describe, it, afterEach, expect, vi } from "vitest";
+import { getCurrentScript, memoize } from "@krakenjs/belter/src";
+
+import {
+  getStorageState,
+  getStorageID,
+  getSessionState,
+  getClientMetadataID,
+} from "./session";
+
+const clientId = "foobar123";
+const mockScriptSrc = `https://test.paypal.com/sdk/js?client-id=${clientId}`;
+
+function makeMockScriptElement(src = mockScriptSrc): HTMLCanvasElement {
```

Comment:
I did this in a few files because it's smaller scope and more understandable/manageable (IMO) than the `insertMockSDK` or whatever fn in src/test.js

But, it's also repetitive. I could put it in a file like `./test/helpers`? Any opinions?